docs/examples/word_embedding/word_embedding_training.md

```diff
@@ -147,11 +137,31 @@ data, batchify_fn, subword_function = transform_data_fasttext(
 batches = data.transform(batchify_fn)
 ```
 
+Note that the number of subwords is potentially
+different for every word. Therefore the batchify_fn represents a word with its
+subwords as a row in a compressed sparse row (CSR) matrix. For more information on CSR matrices click here:
+https://mxnet.incubator.apache.org/tutorials/sparse/csr.html
+
+Separating the batchify_fn from the previous word-pair
+sampling is useful, as it allows parallelization of the CSR matrix construction over
+multiple CPU cores for separate batches.
+
+##### Citations:
+
+- [1] Mikolov, Tomas, et al. “Distributed representations of words and phrases
+and their compositionally.”
+   Advances in neural information processing
+systems. 2013.
+
+
+- [2] Bojanowski et al., "Enriching Word Vectors with Subword
+Information" Transactions of the Association for Computational Linguistics 2017
+
 ## Subwords
 
-`gluonnlp` provides the concept of a SubwordFunction which maps
+`GluonNLP` provides the concept of a Subword function which maps
 words to a list of indices representing their subword.
-Possible SubwordFunctions
+Possible Subword functions
```

Comment:
subword*, to match the lower-casing elsewhere

Reply:
same comment as above